src/status_im/contexts/wallet/bridge/input_amount/view.cljs

```diff
@@ -10,17 +10,14 @@
   []
   [rn/view {:style style/bridge-send-wrapper}
    [input-amount/view
-    {:current-screen-id      :screen/wallet.bridge-input-amount
-     :button-one-label       (i18n/label :t/review-bridge)
-     :button-one-props       {:icon-left :i/bridge}
-     :enabled-from-chain-ids (rf/sub
```

Comment:
`enabled-from-chain-ids` and `from-enabled-networks` don't need to be an argument, they turned into subscriptions `:send-input-amount-screen/enabled-from-chain-ids` and `:send-input-amount-screen/from-enabled-networks` respectively.